lib/include/tick/prox/prox_binarsity.h

```diff
@@ -27,19 +33,35 @@ class DLL_PUBLIC TProxBinarsity : public TProxWithGroups<T> {
                  SArrayULongPtr blocks_length, ulong start, ulong end,
                  bool positive);
 
-  TProxBinarsity() = delete;
-  TProxBinarsity(const TProxBinarsity& other) = delete;
-  TProxBinarsity(TProxBinarsity&& other) = delete;
-  TProxBinarsity& operator=(const TProxBinarsity& other) = delete;
-  TProxBinarsity& operator=(TProxBinarsity&& other) = delete;
```

Comment:
Are they unnecessary now? I thought it was best practice to set these.

Reply:
IMO it is, but unless we're going to put them everywhere I don't see the point.

I had added them while I was trying to figure out what was wrong with building on windows. it turned out the entire source of ProxWithGroups had to in the header

Reply:
Ok your choice, I wouldn't have even thought to add them :)!

Reply:
https://cpppatterns.com/patterns/rule-of-five.html